commands/secret_list.go

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) OpenFaaS Author(s) 2018. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+package commands
+
+import (
+	"bytes"
+	"fmt"
+	"os"
+	"text/tabwriter"
+
+	"github.com/openfaas/faas-cli/proxy"
+	"github.com/openfaas/faas-cli/schema"
+	"github.com/spf13/cobra"
+)
+
+// secretListCmd represents the secretCreate command
+var secretListCmd = &cobra.Command{
+	Use:   "list",
```

Comment:
I think it's good to have the `ls` alias as well, because it's common for most commands and people are used to using it